common/buildcraft/energy/BCEnergyConfig.java

```diff
@@ -82,13 +154,13 @@ public static void preInit() {
             "minecraft:hell", //
             "minecraft:sky",//
         };
-        propExcludedBiomes = BCCoreConfig.config.get("worldgen", "excludedBiomes", _excluded);
+        propExcludedBiomes = BCCoreConfig.config.get("worldgen.oil", "excludedBiomes", _excluded);
         propExcludedBiomes.setComment("Biome registry names (e.g. 'minecraft:hell','minecraft:jungle')"
             + " of biomes that should never generate oil.");
         world.setTo(propExcludedBiomes);
 
         int[] _dims = { -1, 1 };
-        propExcludedDimensions = BCCoreConfig.config.get("worldgen", "excludedDimensions", _dims);
+        propExcludedDimensions = BCCoreConfig.config.get("worldgen.oil", "excludedDimensions", _dims);
```

Comment:
Please don't change the names - you'll break existing configs.

Either add a comment to https://github.com/BuildCraft/BuildCraft/issues/4272 to do this later *or* move the existing config section to the new one. (I haven't done it before but I think it's possible)

Reply:
They're now moved I believe